app/controllers/admin/principals_controller.rb

```diff
@@ -8,6 +8,19 @@ def show
     @principal = principal
   end
 
+  def destroy
```

Comment:
Sure the entire `/admin` route is wrapped in HTTP auth:

https://github.com/moneyadviceservice/rad/blob/master/config/routes.rb#L81

![screen shot 2016-02-26 at 15 54 15](https://cloud.githubusercontent.com/assets/306583/13357296/7d3ac81a-dca1-11e5-8b06-e29f530c606c.png)